test/test-run-express-metrics.js

```diff
@@ -0,0 +1,35 @@
+var helper = require('./helper');
```

Comment:
The code starting sl-run and listening for messages is based on what was in `test/test-run-metrics.js`. IIUC, the messages are send via `fd 3` of type `ipc`, so there shouldn't be any race condition?